pkg/sql/codegen/couler/codegen_test.go

```diff
@@ -75,3 +77,82 @@ INTO sqlflow_models.my_xgboost_model;
 			Label: &ir.NumericColumn{&ir.FieldMeta{"class", ir.Int, "", []int{1}, false, nil, 0}}},
 	}
 }
+
+var testCoulerClusterConfig = `
+class K8s(object):
+    def __init__(self):
+        pass
+
+    def with_pod(self, template):
+        self._with_medadata(template)
+        self._with_tolerations(template)
+        return template
+
+    def _with_medadata(self, template):
+        template["metadata"] = {}
+        template["metadata"]["namespace"] = "sqlflow"
```

Comment:
It seems that we can not specify `namespace` here, it's not the workflow namespace but Pod template namespace.

Reply:
We can add the `namespace` field in the `kubectl` configuration file as the default namespace.